<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>511</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777213</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>355</width>
     <height>130</height>
    </size>
   </property>
   <widget class="QPushButton" name="btnLoadOriginalJson">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>27</y>
      <width>135</width>
      <height>26</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Load original JSON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoadFinalJson">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Load final JSON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShowOriginalJson">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>77</y>
      <width>100</width>
      <height>26</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Show original</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShowFinalJson">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Show final</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRun">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>321</width>
      <height>26</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>RUN LOADING CARTS</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbGroupingAlg">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>279</width>
      <height>26</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="placeholderText">
     <string>Choose grouping customer algorithm</string>
    </property>
    <item>
     <property name="text">
      <string>AVG DISTANCES METHOD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LEAST SQUARE METHOD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>K MEANS CLUSTERING</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AUTO</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cmbBPAlgorithm">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>127</y>
      <width>354</width>
      <height>26</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="placeholderText">
     <string>Choose Bin Packing algorithm</string>
    </property>
    <item>
     <property name="text">
      <string>1D_FFD_BASE_LINE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1D_FFD_ITEMS_SPLITTING</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1D_FFDS_MULTI_BOXES</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3D_LIBRARY_ONE_BOX</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3D_LIBRARY_FFDSEnhancement_MULTI_BOXES</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3D_LIBRARY_MULTI_BOXES_COUNT_DEFINED</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cmbTSPAlg">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>227</y>
      <width>178</width>
      <height>26</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="placeholderText">
     <string>Choose TSP algorithm</string>
    </property>
    <item>
     <property name="text">
      <string>NEAREST NEIGHBOR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GREEDY</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NONE</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cmbNumKMeansIter">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>180</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>200</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="whatsThis">
     <string>Number of K-Means iterations</string>
    </property>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>40</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>60</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="txtBoxHeight">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtBoxDepth">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>340</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>370</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>ADD BOX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemoveBoxes">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>400</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>REMOVE BOXES</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Width:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Height:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Depth:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Max-Weight:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtBoxMaxWeight">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtBoxName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtBoxCount">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::WidgetWidth</enum>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Count:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWrong">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>470</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CHYBA!!!!</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtBoxWidth">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::WidgetWidth</enum>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
